Project.toml

```diff
@@ -43,6 +43,7 @@ SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
`@test_logs` comes from `Test` so I don't think this is needed?

Reply:
Hm, must have missed that when testing (probably tried without `using Test`) :)
Will remove

Reply:
Edit: It seems actually to be required for `min_level=Logging.Warn`